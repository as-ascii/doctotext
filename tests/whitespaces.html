<!DOCTYPE html>
<html>
<body>

<h1>Test of significant whitespaces.</h1>

<p>Two spaces between digits: 1&nbsp;&nbsp;2<br />
Three spaces between digits: 1&nbsp;&nbsp;&nbsp;2<br />
Whitespace at the beginning of the next line:<br />&nbsp;1<br />
Tab between digits: 1&#9;2<br />
Two tabs between digits: 1&#9;&#9;2<br />
Tab at the beginning of the next line:<br />&#9;1<br />
Line break (shift+enter) between digits: 1<br />
2</p>

</body>
</html>
