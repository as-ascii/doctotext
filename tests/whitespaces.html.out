Test of significant whitespaces.
Two spaces between digits: 1  2
Three spaces between digits: 1   2
Whitespace at the beginning of the next line:
 1
Tab between digits: 1	2
Two tabs between digits: 1		2
Tab at the beginning of the next line:
	1
Line break (shift+enter) between digits: 1
2

